sources/au/qld/statewide.json

```diff
@@ -21,28 +21,27 @@
                     "text": "CC BY 4.0",
                     "url": "http://creativecommons.org/licenses/by/4.0/",
                     "attribution": true,
-                    "attribution name": "© State of Queensland (Department of Natural Resources, Mines and Energy) 2021",
+                    "attribution name": "© State of Queensland (Department of Resources) 2024",
                     "share-alike": false
                 },
                 "note": "must use web interface & email workflow to download fresh data.",
                 "conform": {
                     "format": "gdb",
                     "file": "DP_QLD_DCDB_WOS_CUR.gdb",
-                    "layer": "QLD_ADDRESS_QAMF",
-                    "unit": "UNIT_NUMBER",
+                    "layer": "QLD_LOCATION_ADDRESS",
+                    "unit": [
+                        "UNIT_TYPE",
```

Comment:
it's probably better to leave out the unit type, as the type is not as important and more likely to confuse data consumers. In a future OA schema it could go into its own field.

![image](https://github.com/openaddresses/openaddresses/assets/117278/d0e65be7-0bc8-4836-92ae-a0b2fb4e5fc8)

Reply:
The "unit" is meant for sub-address stuff like this. We want to get "APT 1" or "SHED" in this field.

Reply:
Yes but I'd just expect `["UNIT_NUMBER", "UNIT_SUFFIX"]` here, otherwise data consumers need to parse out the prefix and need to implement rules specific to each OA source since what these prefixes mean is specific to the source. For a map, you'd likely want to omit the prefix and label like "1A" but for a full postal address you'd want the prefix, eg. "Unit 1A", expanding the "U" into "Unit".

Reply:
I think it's still useful to include UNIT_TYPE here, otherwise you can't tell the difference between "APT 1  A" and "U 1 A" or "VLLA 1 A". I agree that the data source's encoding/shortening in "UNIT_TYPE" isn't ideal, but it's still useful and important information to include in our output.

If we know what the encodings are via documentation, we could use a conform function to expand them, I think.